driver-async/src/main/com/mongodb/async/client/internal/CommandMarker.java

```diff
@@ -50,6 +50,13 @@
             connectionString = "mongodb://localhost:27020";
         }
 
+        if (!options.containsKey("mongocryptdBypassSpawn") || !((Boolean) options.get("mongocryptdBypassSpawn"))) {
+            processBuilder = new ProcessBuilder(createMongocryptdSpawnArgs(options));
+            startProcess();
```

Comment:
Start the process in the constructor.  Now that we're doing this, also moving MongoClient construction below to make it less likely that it will log a failure to connect to connect (which the driver does at INFO level).

Reply:
Nice